packages/fxa-settings/src/index.tsx

```diff
@@ -1,11 +1,34 @@
-import React from "react";
-import ReactDOM from "react-dom";
-import "./index.css";
-import App from "./App";
-
-ReactDOM.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
-  document.getElementById("root")
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { render } from 'react-dom';
+import './index.scss';
+import App from './components/App';
+import config from './lib/config';
+import sentryMetrics from './lib/sentry';
+
+async function init() {
+  if (config.sentry.dsn) {
+    sentryMetrics.configure(config.sentry.dsn, config.version);
```

Comment:
Does this also fix #5017 then? Part of that one was checking on if `AppErrorBoundary` could/should be a shared component (hence the est of 3), I know we said for now let's not do that, but I also partially told you that because I figured we could look into it when we actually set up Sentry, which I didn't know you'd be doing in this PR. 😂 Is it a headache to pass in the instance / is it better separated or do you think we should comment on #5017 and say it's done except for looking further into sharing?

Reply:
Check out my latest push - I have sharing set up I _think_.